packages/dev-server/src/main.ts

```diff
@@ -183,7 +183,7 @@ app.all('/:routeName', async (req, res) => {
   }
 })
 
-const reloadLambdas = (path: string) => {
+const reloadLambdas = (_path: string) => {
```

Comment:
I don't seem to be using that at all.
```suggestion
const reloadLambdas = () => {
```